### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rgibbard**

### CODE REVIEW

File: src/oracledb_datapump/files.py [hunk 1]
```diff
@@ -852,6 +852,8 @@ def resolve_sequences_for_import(
                 resolved_files,
             )
 
+        logger.debug("Resolved dumpfiles for import: %s, ")
+
```
Comment (raginjason) [inline src/oracledb_datapump/files.py:856]: This seems to be missing an actual variable

File: src/oracledb_datapump/files.py [hunk 2]
```diff
@@ -878,6 +880,8 @@ def resolve_sequences_for_export(
                 sequenced_file = make_seq_dumpfile(file.name, seq_num)
                 resolved_files.add(sequenced_file)
 
+        logger.debug("Resolved dumpfiles for export: %s, ")
+
```
Comment (raginjason) [inline src/oracledb_datapump/files.py:884]: This seems to be missing an actual variable

**Review by raginjason — CHANGES_REQUESTED** [review]

File: src/oracledb_datapump/files.py (hunk 1 above)
Comment (rgibbard) [reply src/oracledb_datapump/files.py]: Good catch. Linter/Pylance never catches that. See revision.

File: src/oracledb_datapump/files.py (hunk 2 above)
Comment (rgibbard) [reply src/oracledb_datapump/files.py]: Good catch. Linter/Pylance never catches that. See revision.

**Review by raginjason — APPROVED** [review]

**Pull request merged**